### DISCUSSION TITLE
Log out redirection

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1232] [Q&A] [answered]
Where can I set a logout redirection in order to have the user taken to the home page when they logout. Currently it leads me to https://dev.cmtmi.com/tabid/0/Default.aspx which does not exist.

**Comment by meetmandeep** [comment]
![image](https://user-images.githubusercontent.com/29958123/118054159-a012e500-b33a-11eb-8e5e-d838700e8b6a.png)
![image](https://user-images.githubusercontent.com/29958123/118054178-a99c4d00-b33a-11eb-8819-024712b5ecf8.png)

**Comment by scottw12** [comment, marked as answer]
Thanks
